README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
```

Comment:
Evaluation HW-01-Sanjana Sarathy
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |           |    X          |            |
| **Ease of access**          |     X      |              |            |

Remarks:

Great work Sanjana. You used all five of the elements for your biography and used them to great effect. I like the way you modified headers to differentiate sizes of the main title of your document as well as the sub-sections in a coherent manner. The coding also looks very neat-evenly spaced out. ㅓJust as a thought about comments you made, the spacing aspect of markdown is really important, and from my experience of Rmarkdowns, this is also true. Also, before you load it up on Github page, there is a way to check how your html output would look, see here for instructions. But to be honest, I find it’s easier to check on Github than going through the procedure of producing an html preview. That might come more useful for other outputs.